src/Mod/Draft/draftgeoutils/faces.py

```diff
@@ -138,13 +138,46 @@ def create_face(w1, w2):
             and len(w1.Edges) > 1
             and len(w1.Edges) == len(w2.Edges)):
         faces = []
+        facesList = []
         for (edge1, edge2) in zip(w1.Edges, w2.Edges):
-            face = create_face(edge1, edge2)
-            if face is None:
-                return None
-            faces.append(face)
+            from draftgeoutils.intersections import findIntersection
+            # Find potential intersection due to ArchWall Align in opposite
+            # direction (or Edge's Orientation in opposite direction,
+            # or a combinantion of the above)
+            #              v
+            # w1 ----------+
+            #              |
+            # w2 ----------+---------- w1
+            #              |
+            #              +---------- w2
+            # TODO Maybe those edge pair should not be generated in offsetWire().
```

Comment:
For a better understanding of the issue it would help if the 2 vertical edges below the v are marked with w1 and w2 as well. It is those edges that the code deals with.